### PR TITLE
Ethan check email signup

### PR DESCRIPTION
**Pull request by EtSubas**
added checking the email before a user is added to the database to ensure no conflicting emails exist

### CODE REVIEW

File: client/src/app/user.service.ts [hunk 1]
```diff
@@ -7,12 +7,14 @@ import { User } from './user';
   providedIn: 'root'
 })
 export class UserService {
-  private url = 'https://armchair-athletes-live-sports-s-ofsyvtifhq-uc.a.run.app';
+  private url = 'http://localhost:5200';
```
Comment (Jgerbrandt) [inline client/src/app/user.service.ts:10]: bozo

File: server/.env
Comment (Jgerbrandt) [inline server/.env:1]: don't push this. massive security issue

File: client/src/app/user.service.ts (hunk 1 above)
Comment (EtSubas) [reply client/src/app/user.service.ts:10]: damn

File: server/.env
Comment (EtSubas) [reply server/.env:1]: i thought i didnt mb

File: client/src/app/user.service.ts (hunk 1 above)
Comment (EtSubas) [reply client/src/app/user.service.ts]: this is fixed

**Review by Jgerbrandt — APPROVED** [review]

**Pull request merged**